src/ipahealthcheck/core/config.py

```diff
@@ -93,6 +93,27 @@ def merge(self, d):
             self.__d[key] = d[key]
 
 
+def convert_string(value):
+    """
+    Reading options from the configuration file will leave them as
+    strings. This breaks boolean values so attempt to convert them.
+    """
+    if not isinstance(value, str):
+        return value
+
+    if value.lower() in (
+        "true",
+        "false",
+    ):
+        return value.lower() == 'true'
```

Comment:
With the convert_string function, `debug=other` still triggers the debug mode. The boolean options should probably be handled separately (with something like `if key in ('debug', 'verbose'):...`)

Reply:
I was trying to avoid that because I can't control what other users of healthcheck may configure as variables (pki has its own pki-healthcheck command). I suppose I can add it for the currently known booleans. I don't think I have access to the argparse options directly at this point so I can't auto-detect the type.